assembly.xml

```diff
@@ -15,6 +13,9 @@
     <fileSet>
       <outputDirectory>.</outputDirectory>
       <directory>${project.build.directory}/${project.artifactId}-${project.version}</directory>
+      <excludes>
+        <exclude>docker-embedded/**</exclude>
```

Comment:
Could you renamed to `bundled-docker`?
And anyhow, must that be specifically excluded? I though we building the images outside of the distro folder?